firefox-ios/firefox-ios-tests/Tests/ClientTests/Sharing/ShareManagerTests.swift

```diff
@@ -396,7 +396,8 @@ final class ShareManagerTests: XCTestCase {
     func testGetActivityItems_forTab_withSentFromFirefoxEnabled_OverridesURL_withTreatmentA() throws {
         setupNimbusSentFromFirefoxTesting(isEnabled: true, isTreatmentA: true)
 
-        let expectedShareContentA = "https://mozilla.org\n\nSent from Firefox 🦊 Try the mobile browser: https://mzl.la/4fOWPpd"
+        let expectedShareContentA =
+            "https://mozilla.org\n\nSent from Firefox 🦊 Try the mobile browser: https://mzl.la/4fOWPpd"
```

Comment:
Not sure why swiftlint complained about these tests unrelated to my work. I may open a new branch instead.